device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/transport/amqps/SendResult.java

```diff
@@ -0,0 +1,45 @@
+/*
+*  Copyright (c) Microsoft. All rights reserved.
+*  Licensed under the MIT license. See LICENSE file in the project root for full license information.
+*/
+
+package com.microsoft.azure.sdk.iot.device.transport.amqps;
+
+/**
+ * All the different results of a send message operation that an AMQP device session can report
+ */
+enum SendResult
+{
+    /**
+     * The message was sent successfully.
+     */
+    SUCCESS,
+
+    /**
+     * The message belonged to a different device session.
+     */
+    WRONG_DEVICE,
+
+    /**
+     * The message was a twin/method subscription message and the device session has already sent that subscription message.
```

Comment:
What does it mean that the device already sent that subscription message? Is this just an echo back?

Reply:
My point is, the comment could be more conceptual in nature, rather than talk about the mechanism by which a subscription on a topic is requested.